libtransmission/announcer.h

```diff
@@ -159,4 +158,6 @@ class tr_announcer_udp
     // @brief process an incoming udp message if it's a tracker response.
     // @return true if msg was a tracker response; false otherwise
     virtual bool handle_message(uint8_t const* msg, size_t msglen) = 0;
+
+    [[nodiscard]] virtual bool is_idle() = 0;
```

Comment:
Can this method be and the impl in libtransmission/announcer-udp.cc:693 be `const`? I might be missing something but it looks like they could be

Reply:
f47c2f368f1633e93894958551de0ce6da31ce35